testing/multi-env-test-engine/src/main/java/org/projectnessie/junit/engine/MultiEnvExtensionRegistry.java

```diff
@@ -66,4 +66,9 @@ public Stream<? extends MultiEnvTestExtension> stream(Class<?> testClass) {
                 .flatMap(registry::stream);
     return r;
   }
+
+  public void clear() {
+    registry = MutableExtensionRegistry.createRegistryWithDefaultExtensions(
+      new DefaultJupiterConfiguration(new EmptyConfigurationParameters()));
```

Comment:
Let's call this from the constructor to avoid duplicating (re-)initialization logic.

Reply:
No longer called in two places